doc/setup.md

```diff
@@ -55,8 +55,19 @@ To use **N-API** in a native module:
 ```gyp
   'defines': [ 'NAPI_DISABLE_CPP_EXCEPTIONS' ],
 ```
-
-  4. Include `napi.h` in the native module code.
+  4. **Only** for **macOS** user please don't forget to insert these additional 
+  settings in the `binding.gyp` file:
+  
+  ```gyp
+  ['OS=="mac"', {
+      'cflags+': ['-fvisibility=hidden'],
+      'xcode_settings': {
+        'OTHER_CFLAGS': ['-fvisibility=hidden']
```

Comment:
Instead of `'OTHER_CFLAGS': ['-fvisibility=hidden']`, you can use `'GCC_SYMBOLS_PRIVATE_EXTERN': 'YES', # -fvisibility=hidden`